service-mesh/enable-service-mesh.sh

```diff
@@ -12,7 +12,33 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 kubectx central
-bash <( gsutil cat gs://anthos-workshop/csm-alpha-onboard.sh )
-kubectl delete po $(kubectl get pod -n istio-system -l app=telemetry -o json | jq -r '.items[].metadata.name') -n istio-system
 
+# Enable the required GCP APIs for the topology view
+gcloud services enable iamcredentials.googleapis.com contextgraph.googleapis.com --project ${PROJECT}
```

Comment:
We're enabling the apis in the workshop doc. Should keep them all together either in the doc or in the scripts

Reply:
Agree, will keep them in the doc.